Core/Core/en.lproj/Localizable.stringsdict

```diff
@@ -188,5 +188,27 @@
             <string>%d need grading</string>
 		</dict>
 	</dict>
+	<key>conversation_recipients_to</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@audience_size@</string>
+		<key>audience_size</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>d</string>
+            <key>one</key>
+            <string>%d other</string>
+            <key>two</key>
+            <string></string>
+            <key>few</key>
+            <string></string>
+            <key>many</key>
+            <string></string>
+            <key>other</key>
+            <string>%d others</string>
+		</dict>
```

Comment:
the next time someone edits this in Xcode it will reformat this indent. Probably would be good to do that in this commit so it isn't noise later.

Reply:
I edited this using xcode and this seems to be the way it formatted it.